client/main.lua

```diff
@@ -31,10 +37,17 @@ RegisterCommand('vol', function(source, args)
 	end
 end)
 
-function toggleVoice(tgtId, enabled)
+function toggleVoice(tgtId, enabled, effectType)
+	if enabled and effectType then
```

Comment:
Possibly add a `if GetConVarInt('voice_enableSubmix', 0) == 1 then` check so server owners can decide if they want to enable submixes or not.

Reply:
yea sure